<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="human_tf" value="human" />
    <param name="served_action_name" type="string" value="/repeat_action" />
    <param name="get_plan_service_name" type="string" value="/move_base/make_plan" />
    <param name="move_base_action_name" type="string" value="/move_base" />
    <param name="odometry_topic" type="string" value="/mobile_base_controller/odom" />
    <param name="human_tf" type="string" value="$(arg human_tf)" />
    
    <node pkg="repeat_action_server" name="repeat_action_server" type="repeat_action_server" output="screen" />
</launch>